newIDE/app/src/Debugger/DebuggerConsole.js

```diff
@@ -0,0 +1,101 @@
+// @flow
+import React from 'react';
+
+import { List, CellMeasurer, CellMeasurerCache } from 'react-virtualized';
+import MUIList from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+
+import Checkbox from '../UI/Checkbox';
+
+import InfoIcon from '@material-ui/icons/Info';
+import WarningIcon from '@material-ui/icons/Warning';
+import ErrorIcon from '@material-ui/icons/Error';
+
+export type Log = {
+  message: string,
+  type: 'info' | 'warning' | 'error',
+  group: string,
+  internal: boolean,
+};
+
+const iconMap = {
+  info: <InfoIcon color="primary" />,
+  warning: <WarningIcon color="secondary" />,
+  error: <ErrorIcon color="error" />,
+};
+
+export const DebuggerConsole = ({ logs }: { logs: Array<Logs> }) => {
+  const sizeMeasurer = React.useRef(null);
+  const cache = React.useRef(
+    new CellMeasurerCache({
+      defaultHeight: 45,
+      minHeight: 30,
+      fixedWidth: true,
+    })
+  ).current;
+
+  const [showGroup, _setShowGroup] = React.useState(true);
+  const setShowGroup = (show: boolean) => {
+    _setShowGroup(show);
+    cache.clearAll();
+  };
+
+  return (
+    <>
+      <Checkbox
+        style={{ margin: 10 }}
+        checked={showGroup}
+        label="Show group names"
+        onCheck={(_, value) => setShowGroup(value)}
```

Comment:
```suggestion
        onCheck={(e, value) => setShowGroup(value)}
```
Because event :)

Reply:
The `_` signifies we don't care about the event though only about the value